sum-an-array/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+  <script>
+
+
+    function sumArray(numbersArray){
+      var sum =0;
+    for(var i=0; i<numbersArray.length; i++)
```

Comment:
While this is functional, I don't ever recommend using a for loop without the curly braces setting up your code block. Please revise this to include the proper for loop syntax 👍